README.md

```diff
@@ -44,49 +44,49 @@ making it much easier to get a running index server.
 
 Table of Contents
 
-- [pypiserver - minimal PyPI server for use with pip/easy_install](#pypiserver---minimal-pypi-server-for-use-with-pipeasy_install)
-  - [Quickstart Installation and Usage](#Quickstart-Installation-and-Usage)
-    - [More details about pypi-server run](#More-details-about-pypi-server-run)
-    - [More details about pypi-server update](#More-details-about-pypi-server-update)
-  - [Client-Side Configurations](#Client-Side-Configurations)
-    - [Configuring pip](#Configuring-pip)
-    - [Configuring easy_install](#Configuring-easy_install)
-  - [Uploading Packages Remotely](#Uploading-Packages-Remotely)
-    - [Apache like Authentication ( htpasswd )](#Apache-like-Authentication)
-    - [Upload with setuptools](#Upload-with-setuptools)
-    - [Upload with twine](#Upload-with-twine)
-  - [Using the Docker Image](#Using-the-Docker-Image)
-  - [Alternative Installation methods](#Alternative-Installation-methods)
-    - [Installing the Very Latest Version](#Installing-the-Very-Latest-Version)
-  - [Recipes](#Recipes)
-    - [Managing the Package Directory](#Managing-the-Package-Directory)
-    - [Serving Thousands of Packages](#Serving-Thousands-of-Packages)
-    - [Managing Automated Startup](#Managing-Automated-Startup)
-      - [Running as a systemd service](#Running-as-a-systemd-service)
-      - [Launching through supervisor](#Launching-through-supervisor)
-      - [Running as a service with NSSM (Windows)](#Running-as-a-service-with-NSSM)
-    - [Using a Different WSGI Server](#Using-a-Different-WSGI-Server)
-      - [Apache](#Apache)
-      - [Gunicorn](#Gunicorn)
-      - [Paste](#Paste)
-    - [Behind a Reverse Proxy](#Behind-a-Reverse-Proxy)
-      - [Nginx](#Nginx)
-      - [Supporting HTTPS](#Supporting-HTTPS)
-      - [Traefik](#Traefik)
-    - [Utilizing the API](#Utilizing-the-API)
-      - [Using Ad-Hoc Authentication Providers](#Using-Ad-Hoc-Authentication-Providers)
-    - [Use with MicroPython](#Use-with-MicroPython)
-    - [Custom Health Check Endpoint](#Custom-Health-Check-Endpoint)
-      - [Configure a custom health check by CLI arguments](#Configure-a-custom-health-check-by-CLI-arguments)
-      - [Configure a custom health endpoint by script](#Configure-a-custom-health-endpoint-by-script)
-    - [Sources](#Sources)
-    - [Known Limitations](#known-limitations)
-    - [Similar Projects](#similar-projects)
-      - [Unmaintained or archived](#unmaintained-or-archived)
-    - [Related Projects](#related-projects)
-    - [License](#license)
-
-## [Quickstart Installation and Usage](#pypiserver)
+* [pypiserver - minimal PyPI server for use with pip/easy_install](#pypiserver---minimal-pypi-server-for-use-with-pipeasy_install)
+  * [Quickstart Installation and Usage](#Quickstart-Installation-and-Usage)
```

Comment:
Random quick idea: If I try to copy [the generated link from the GitHub UI](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links), I get the `#` reference in all lower case, do you think this could be worth trying out? 🤔

Example:
>
> <img width="350" alt="image" src="https://github.com/pypiserver/pypiserver/assets/12134172/959a8b16-13b8-4b1c-929b-b49c9f0bf39f">
 >
>
> generates: 
>`https://github.com/pypiserver/pypiserver/blob/17e5304a1b32c21e18cbbe7c7f583bc41bf0b40c/README.md#more-details-about-pypi-server-run `

Reply:
I just fixed the mdformat grumble and will look into you suggestion. With those changes at https://github.com/ghinks/pypiserver with the PR branch set as the primary branch the TOC links do appear to be working ( I"m using brave browser ).

Reply:
Oh great! I've tried it out on https://github.com/ghinks/pypiserver with Chrome and it also seems to be working 👍